packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -3,7 +3,9 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io';
```

Comment:
we don't import `dart:io` directly in flutter_tools; instead, import `'../../base/io.dart'`

Reply:
Sounds good, will fix 👍